FastAnimations/README.md

```diff
@@ -29,6 +29,8 @@ normal speed or `2` for double speed).
   `FishingSpeed`       | 1×      | How fast you cast and reel when fishing (doesn't affect the minigame).<br /><small>(Suggested value: 2×.)</small>
   `MilkSpeed`          | 5×      | How fast you use the milk pail.
   `ShearSpeed`         | 5×      | How fast you use the shears.
+  `ToolSwingSpeed`     | 1×      | How fast you swing your tools (excludes weapons & the fishing rod).<br /><small>(This could be considered cheating as it can make hoeing/chopping trees/breaking rocks much quicker)</small>
```

Comment:
Many of the disabled-by-default options could be considered cheating, so there's no need to mention it here.